libzecale/circuits/aggregator_circuit_wrapper.hpp

```diff
@@ -25,19 +25,20 @@ class aggregator_circuit_wrapper
 private:
     using wsnark = typename wverifierT::snark;
 
-    std::shared_ptr<
-        aggregator_gadget<nppT, wppT, nsnarkT, wverifierT, NumProofs>>
-        aggregator_g;
+    const size_t num_inputs_per_nested_proof;
+
+    // TOOD: Cache the circuit to save reconstructing it at every call.
```

Comment:
TODO (that's a detail, but that's helpful to fix this typo as I sometimes grep all `TODO` in the code base to track down what needs to be done and may have been forgotten)

Reply:
Yeah, good catch.  Fixed. Thanks.